internal/collector/nginxplusreceiver/scraper.go

```diff
@@ -68,12 +78,14 @@ func newNginxPlusScraper(
 	}
 
 	return &nginxPlusScraper{
-		plusClient: plusClient,
-		settings:   settings,
-		cfg:        cfg,
-		mb:         mb,
-		rb:         rb,
-		logger:     settings.Logger,
+		plusClient:                    plusClient,
+		settings:                      settings,
+		cfg:                           cfg,
+		mb:                            mb,
+		rb:                            rb,
+		logger:                        settings.Logger,
+		previousLocationZoneResponses: make(map[string]ResponseStatuses),
```

Comment:
Can we call the plus api here to populate the `previousLocationZoneResponses` & `previousServerZoneResponses` maps before we start scraping?